UIKit+Combine/UITableView+Combine.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  UITableView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 19/01/20.
+//  Copyright © 2020 Joan Disho. All rights reserved.
+
+import Foundation
+import UIKit
+import Combine
+
+private class TableViewDelegateProxy: DelegateProxy, UITableViewDelegate, DelegateProxyType {
+    func setDelegate(to object: UITableView) {
+        object.delegate = self
+    }
+}
+
+extension UITableView {
```

Comment:
like mentioned above it might be nicer to make the extension `public` and remove it from all the standalone publishers (also relocate this to the top etc)